packages/swirl-components/src/components/swirl-date-picker/swirl-date-picker.tsx

```diff
@@ -1,97 +1,63 @@
 import {
   Component,
+  Element,
   Event,
   EventEmitter,
   h,
   Host,
   Prop,
-  Watch,
 } from "@stencil/core";
-import AirDatepicker, {
-  AirDatepickerDate,
-  AirDatepickerLocale,
-} from "air-datepicker";
+import { WcDatepickerCustomEvent } from "wc-datepicker/dist/types/components";
+import { WCDatepickerLabels } from "wc-datepicker/dist/types/components/wc-datepicker/wc-datepicker";
 
-import localeEn from "air-datepicker/locale/en";
+import "wc-datepicker";
 
 @Component({
   shadow: true,
   styleUrl: "swirl-date-picker.css",
   tag: "swirl-date-picker",
 })
 export class SwirlDatePicker {
-  @Prop() locale?: Partial<AirDatepickerLocale> = localeEn;
-  @Prop() maxDate?: Date;
-  @Prop() minDate?: Date;
+  @Element() el: HTMLElement;
+
+  @Prop() labels?: WCDatepickerLabels;
+  @Prop() locale?: string = "en-US";
```

Comment:
**nitpick:** in the story prop it shows string as prop type but lets you enter an object.
<img width="751" alt="image" src="https://user-images.githubusercontent.com/23700195/212655058-03215576-2996-4e2a-9f6d-dcd9d06f850b.png">

Reply:
Fixed.